src/main/java/com/xiaomi/infra/pegasus/client/PegasusTable.java

```diff
@@ -110,7 +112,7 @@ public void onCompletion(client_operator clientOP) {
           public void onCompletion(client_operator clientOP) {
             rrdb_get_operator gop = (rrdb_get_operator) clientOP;
             if (gop.rpc_error.errno != error_code.error_types.ERR_OK) {
-              handleReplicaException(promise, op, table, timeout);
+              handleReplicaException(new Request(hashKey, sortKey), promise, op, table, timeout);
```

Comment:
Maybe you don't need change handleReplicaException, you can get request from op.

Reply:
`op` can‘t restore the `Key` and `value`.